mappings/net/minecraft/client/render/block/entity/BlockEntityRendererFactory.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_5614 net/minecraft/client/render/block/entity/BlockEntityRendererFactory
-	CLASS class_5615 Arguments
+	METHOD create (Lnet/minecraft/class_5614$class_5615;)Lnet/minecraft/class_827;
+		ARG 1 ctx
+	CLASS class_5615 BlockEntityRenderDispatcherContext
```

Comment:
Could this be just `Context`?

Reply:
The `EntityRendererFactory` nested class was already named `EntityRenderDispatcherContext` but I support just `Context`